security/wi-secrets/manifests/bq-readonly-key

```diff
@@ -0,0 +1 @@
+key=my-api-key
```

Comment:
Is this supposed to be a YAML file maybe? If so maybe let's add a .yaml suffix to the file name.

Reply:
Could we also have a comment above the key stating e.g.:
```
# It is not recommended practices to check in secrets into a source code repository
```

Reply:
No it's meant to just be a file. Do you know if the `--data-file` flag in kubectl will accidentally parse the comment as well though ?

Reply:
Nah you're right, it's meant to just be a file, let's leave as-is :)